packages/conf-flint/conf-flint.3.0/opam

```diff
@@ -0,0 +1,47 @@
+opam-version: "2.0"
+maintainer: "bobot"
+homepage: "http://flint.org"
+bug-reports: "https://github.com/flintlib/flint2.git"
+license: "LGPL-2.1-only"
+build: [
+  ["sh" "-exc" "printf \"#include  <flint/flint.h>\\n#if __FLINT_VERSION < 3\\n#error \\\"Flint (>= 3.0) is not accessible\\\"\\n#endif\" > test.c"]
+  ["sh" "-exc" "cc -c $CFLAGS -I/usr/local/include test.c"] {os != "macos" & os != "win32"}
+  [
+		"sh"
+		"-exc"
+		"$(ocamlc -config-var c_compiler) -c $CFLAGS -I/usr/local/include test.c"
+	] {os = "win32" & os-distribution = "cygwinports"}
+  [
+    "sh"
+    "-exc"
+    "cc -c $CFLAGS -I/opt/homebrew/include -I/opt/local/include -I/usr/local/include test.c"
+  ] {os = "macos"}
+]
+depexts: [
+  ["libflint-dev"] {os-distribution = "debian"}
+  ["libflint-dev"] {os-distribution = "ubuntu" }
+  ["flint"] {os = "macos" & os-distribution = "homebrew"}
+  ["flint"] {os-distribution = "macports" & os = "macos"}
+  ["flint" "flint-devel"] {os-distribution = "centos"}
+  ["flint" "flint-devel"] {os-distribution = "fedora"}
+  ["flint" "flint-devel"] {os-distribution = "ol"}
+  ["flint-dev"] {os-distribution = "alpine"}
+  ["flint-devel"] {os-family = "opensuse"}
+  ["libflint-devel"] {os = "win32" & os-distribution = "cygwinports"}
+  ["flint"] {os-distribution = "nixos"}
+  ["flint"] {os-distribution = "arch"}
+]
+available: [
```

Comment:
I am also ok in letting these distributions fail in the CI. Users can still install flint3 manually and then install this

Reply:
Sounds good!